physics/oblate_body_body.hpp

```diff
@@ -28,7 +29,7 @@ OblateBody<Frame>::OblateBody(
     Length const& radius,
     Vector<double, Frame> const& axis)
     : OblateBody(mass,
-                 -j2 * mass * GravitationalConstant * radius * radius,
+                 -j2 * mass * constants::GravitationalConstant * radius * radius,
```

Comment:
Same comment, please add a `using constants::GravitationalConstant` in namespace principia (but before the subnamespace physics).